soc/arc/snps_smic_huangshan/soc_irq.h

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2018 Synopsys, Inc. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _HUANGSHAN_SOC_IRQ_H_
```

Comment:
Are these interrupt vector defines required? Duplicates the dts info, use from there?
Also, please align DTS structure/definition style to latest in #11507